system-tests/README.md

```diff
@@ -57,8 +57,8 @@ System tests are run both in local deployment (using docker compose) and in the
 > For Example:
 
 ```bash
-# EDC (in the EDC root folder)
-git checkout 3ff940b720f44826df28e893fb31344eb6faacef
+# EDC (in the EDC root folder, checkout the release tag to a local branch)
```

Comment:
Why creating a new branch locally? Why only for EDC and not the other repos?

Reply:
because when you checkout a `tag` one must specify a branch name